### PR TITLE
Flash-Based CSRF updates

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/template.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Steps to Reproduce
+
+1. Enable a HTTP interception proxy, such as Burp Suite or OWASP ZAP
+1. Use a browser to sign into the application at: {{URL}}
+1. Navigate to the following URL and submit the form: {{URL}}
+1. Use the HTTP interception proxy to intercept the request triggered by the form
+1. Observe that a request is sent using `.SWF` file:
+
+{{screenshot}}
+
```
Comment (evildaemond) [inline submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/template.md:22]: This seems redundant for triage, at the end of the day, we only require the SWF file, and sometimes a link, not the creation.

**Review by evildaemond — CHANGES_REQUESTED** [review]
1 Comment the LGTM

File: submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/template.md]: This has been addressed, ty

**Review by evildaemond — APPROVED** [review]
LGTM

**Pull request merged**